docs/guides/integration-javascript.md

```diff
@@ -0,0 +1,293 @@
+---
+title: JavaScript Express Integration
+sidebar_label: JavaScript Express Integration
+pagination_label: JavaScript Express Integration
+description: Information about integrating with UID2 SDK for JavaScript as part of your UID2 implementation.
+hide_table_of_contents: false
+sidebar_position: 04
+---
+
+# JavaScript Express Integration Guide
+
+This guide is for publishers who want to integrate with UID2 and generate [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) (advertising tokens) using only JavaScript/TypeScript client-side changes on their website with minimum effort.
+
+This guide does not apply to publishers who want to use a [private operator](../ref-info/glossary-uid.md#gl-private-operator), or want to generate tokens server-side. Those publishers should follow the [UID2 SDK for JavaScript Integration Guide](publisher-client-side.md).
+
+UID2 provides a UID2 SDK for JavaScript (TODO add links!!!) with the following features:
+
+- UID2 token generation
+- Automatic refreshing of UID2 tokens
+- Automatic storage of UID2 tokens in the browser
+
+You'll need to complete the following steps:
+
+1. [Complete UID2 account setup](#complete-uid2-account-setup)
+2. [Add UID2 SDK For JavaScript to Your Site](#add-js-to-your-site)
+3. [Configure the UID2 SDK For JavaScript](#configure-the-uid2-sdk-for-javascript)
+4. [Retrieving the UID2 Advertising Token](#retrieving-the-uid2-advertising-token)
+
+
+## UID2 SDK for JavaScript Version
+
+Support for client-side token generation is available in version 3.2 and above of the SDK. 
+
+The URL for the SDK is https://cdn.prod.uidapi.com/uid2-sdk-3.2.0.js . In the following code examples, we will use the placeholder ```{{ UID2_JS_SDK_URL }}``` to refer to this URL.
+
+If you want to use a debug build of the SDK, use the following URL instead: https://cdn.integ.uidapi.com/uid2-sdk-3.2.0.js . 
+
+## Sample Implementation Website
+
+For an example application, see this example:
+- Running site: [UID2 SDK Integration Example](https://cstg-integ.uidapi.com/)
```

Comment:
A link to the code would be good as well, especially as the example site doesn't have a link to it.